pgrx-tests/src/framework.rs

```diff
@@ -315,6 +314,14 @@ fn install_extension() -> eyre::Result<()> {
         .stderr(Stdio::inherit())
         .env("CARGO_TARGET_DIR", get_target_dir()?);
 
+    if requires_runas() {
+        // if we're running tests as a different operating-system user we actually need to
```

Comment:
Minor legibility nit
```suggestion
        // if we're running tests as a different operating-system user, then we actually need to
```